ContactARep.html

```diff
@@ -29,8 +29,53 @@ <h1 class="h1Other">Contact a Rep</h1>
   </header>
 
   <div class="main">
-    <div>
-    </div>
+      <form action="#">
+        <label>Full Name</label>
+        <input type="text" placeholder="Full Name" name="name" required />
+
+        <br/>
```

Comment:
luke used br's for demo, but consider trying to style forms without them in the future. use CSS and wrappers as needed to make these things line up properly. just setting label and input to have `display: block` will accomplish roughly the same thing